admin-manual/maintenance/web-analytics.rst

```diff
@@ -30,22 +30,96 @@ These types of tools require access to the log files generated by the web server
 Page tagging tools
 ==================
 
-The tools found under this section require only that a system administrator or developer
-insert a JavaScript code snippet into the relevant page int AtoM where you'd
+The tools found under this section require a system administrator or developer
+to insert a JavaScript code snippet into the relevant page in AtoM where you'd
 like to gather analytic data. Probably the best known open source solution is
 `Piwiki <http://piwik.org/>`_.
 
 You should be able to configure any solution in AtoM by editing the
 corresponding `PHP templates <https://github.com/artefactual/atom/tree/2.x/apps/qubit/templates>`_.
 
-As of AtoM 2.0.0, you can configure Google Analytics just adding your tracking
-ID to the app.yml configuration file. Open your favorite text editor and add the
-following contents in it:
+Google Analytics
+++++++++++++++++
+
+As of AtoM 2.0.0, you can configure `Google Analytics`_ by adding your tracking
+ID to the file config/app.yml:
 
 .. code-block:: yaml
 
-   all:
      google_analytics_api_key: UA-XXXXX-X
 
 Replace UA-XXXXX-X with your tracking ID. Once you are done, remember to
 :ref:`clear the cache <maintenance-clear-cache>`.
+
+Google Analytics for tracking institutional pageviews
++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+As of AtoM 2.5, it is possible to track pageviews for individual institutions
+who are sharing a multi-repository AtoM instance by adding a custom dimension to
+the tracking ID. Adding this dimension will set the authorized form of name from
+the repository as the dimension value.
+
+* Description pages (all templates) use the repository value (direct or inherited)
+* Actor pages use the maintaining repository value
+* Repository pages use the repository
+
+Follow the instructions provided in Google's Analytics Help pages to
+`Set up custom dimension`_. When you reach step 6, *Select the scope*, choose
+**Hit** as the scope and set it to *active*. Once you have completed all the
+steps, the custom dimension will be displayed in a table. The index number is
+displayed in the column **Index**.
+
+In the file config/app.yml, set the newly-created dimension index number below
+the API key and make sure that the setting is uncommented.
+
+.. code-block:: yaml
+
+     google_analytics_api_key: UA-XXXXX-X
+     google_analytics_institutions_dimension_index: 1
+
+Remember to clear the cache and restart PHP-FPM after configuration.
```

Comment:
Not sure if we have specific section(s) in the doc to link to explain how to do this, but it'd be nice if we have them.